dev/indexing_test.clj

```diff
@@ -68,7 +68,7 @@
 
   @(fluree/query (fluree/db ledger)
                  {:select {'?s [:* {:schema/isBasedOn [:*]}]}
-                  :where  [['?s :rdf/type :schema/Movie]]})
+                  :where  [['?s :type :schema/Movie]]})
```

Comment:
`rdf:type` or `@type` should work the same for a query - so these shouldn't need changing, but it shouldn't hurt to to change them either.

It is in the responses, when doing a graph crawl specifically, that we should default to returning `@type` instead of defaulting to `rdf:type`.

SPARQL will parse near directly into FlureeQL, and @type doesn't exist in SPARQL... so we don't want to loose rdf:type. SPARQL does not support a graph crawl response, which is where we are in unique Fluree land... and that is why it makes sense to change the default behavior there.

Reply:
Good point, I've added an explicit test that shows the following:
1) you cannot transact with an `rdf:type` predicate
2) you can query with an `rdf:type` predicate and it behaves the same as using `@type`